Formula/mytop.rb

```diff
@@ -22,6 +22,9 @@ class Mytop < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d9e6242116e773721a6a5e8beb39c5936d6f986043118f3216a750cd3d777dfc"
   end
 
+  # https://tracker.debian.org/news/1400605/removed-191-5-from-unstable/
+  disable! date: "2023-06-24", because: "debian patches no longer applied"
```

Comment:
Let's give at least a week's notice of deprecation.
```suggestion
  disable! date: "2023-07-01", because: "debian patches no longer applied"
```

Reply:
Even then I'm not sure if this is really exceptional enough to skip the 3 month minimum deprecation we have. We could probably align the disable date with OpenSSL 1.1 EOL.

I believe MariaDB ships with an updated version `mytop`, so we could point people to using that.

Reply:
Or launchpad actually: https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/mytop/1.9.1-2/mytop_1.9.1-2.debian.tar.xz

Though I would consider an update to https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/mytop/1.9.1-5/mytop_1.9.1-5.debian.tar.xz and add `14_fix_deprecated_show_innodb_status.patch` to the list.

I still think it's worth deprecating as being no longer maintained as well however.

Reply:
#134971